### ISSUE TITLE
PLEASE ADD THE OPTION TO MATCH THE DEEPFAKED FACIAL SKIN TONE WITH FULL BODY SKINTONE. (Especially With the hands) 

### ISSUE THREAD

**Issue by Bobby606** [hacksider/Deep-Live-Cam#180]
@hacksider I really appreciate your work and you have taken deepfaking to the next level with just a single photo even surpassing those other deepfake tool the need hours days or even week to train a model. Deep live cam can do it with a single photo.

But there is one thing that every deepfake lacks. The ability  to match facial skintone to the body skintone especially the hands. For example, with all current deepfakes someone with light skintone can only deepfake a celebrity with light skintone accurately, and also someone with a darker skintone can only deepfake a celebrity with a darker skintone accurately.

But if someone with a lighter skintone want to deepfake a celebrity with a darker skintone the difference is noticeable between the face and the body especially the hands (especially when you are waving your hands to say "hi" on camera) same also goes for someone with a darker skintone trying to deepfake a celebrity with a lighter skintone.

It would be great if there is an option (like a button or slider) to automatically match the skintone or color of the deepfaked face with skintone or color of the visible body part.

In other words the should be a feature or button in the deep live cam software that can change the color of the person's visible body parts to match the color of the deepfaked celebrity face.

If you can possibly integrate this feature into deep live cam. It will be an excellent feature will rank in far above all other deepfakes today.

**Comment by hacksider** [comment]
This is more of a request rather than an issue of the code. Sad to say, most deepfake with single image relies on insightface model.

**Comment by Bobby606** [comment]
> This is more of a request rather than an issue of the code. Sad to say, most deepfake with single image relies on insightface model.

Are you saying it's not possible?
